app/templates/_instructions.html

```diff
@@ -0,0 +1,174 @@
+{% macro r(dataset) -%}
+<p>In order to use Data Package in R follow instructions below:</p>
+<div class="highlight">
+<pre>
+<span></span>install.packages<span class="p">(</span><span class="s">"devtools"</span><span class="p">)</span>
+<span class="kn">library</span><span class="p">(</span>devtools<span class="p">)</span>
+install_github<span class="p">(</span><span class="s">"hadley/readr"</span><span class="p">)</span>
+install_github<span class="p">(</span><span class="s">"ropenscilabs/jsonvalidate"</span><span class="p">)</span>
+install_github<span class="p">(</span><span class="s">"ropenscilabs/datapkg"</span><span class="p">)</span>
+
+<span class="c1">#Load client</span>
+<span class="kn">library</span><span class="p">(</span>datapkg<span class="p">)</span>
+
+<span class="c1">#Get Data Package</span>
+datapackage <span class="o">&amp;</span>lt<span class="p">;</span>
+<span class="o">-</span> datapkg_read<span class="p">(</span>
+<span class="s">"https://bits.datapackaged.com/metadata/{{ dataset.owner }}/{{ dataset.name }}/_v/latest"</span><span class="p">)</span>
+
+<span class="c1">#Package info</span>
+<span class="kp">print</span><span class="p">(</span>datapackage<span class="p">)</span>
+
+<span class="c1">#Open actual data in RStudio Viewer</span>
+{% for resource in dataset.resources %}
+View<span class="p">(</span>datapackage<span class="o">$</span>data<span class="o">$</span><span class="s">"{{resource.name}}"</span><span class="p">)</span>
+{% endfor %}
+</pre>
+</div>
+{%- endmacro %}
+
+{% macro pandas(dataset) -%}
+<p>Tested with Python 3.5.2</p>
+
+<p>To generate Pandas data frames based on JSON Table Schema descriptors we have to install <code>jsontableschema-pandas</code> plugin.
+To load resources from a data package as Pandas data frames use <code>datapackage.push_datapackage</code> function.
+*Storage* works as a container for Pandas data frames.</p>
```

Comment:
is this supposed to be italic? cause it actually  is not Eg:
![image](https://cloud.githubusercontent.com/assets/15143785/25886854/8f9cf19e-3570-11e7-8988-0e47fe49baff.png)